doc/expressionLanguage.md

```diff
@@ -106,16 +107,51 @@ level: 85.3
 The only expression rule that will be executed will be that of the 'fillingLevel' attribute. It will produce the value
 '0.853' that will be sent to the Context Broker.
 
+Note that expressions are only applied if the attribute name (as received by the IOT Agent in the southbound interface) matches the expression variable. Otherwise, the southbound value is used directly. Let's illustrate with the following example:
+```
+  "consumption": {
+   "type": "String",
+   "value": "${trim(@spaces)}"
+ }
+```
+
+- Case 1: the following measure is received at the southbound interface:
+```
+consumption: "0.44"
```

Comment:
L120 uses "0.44" but L133 uses 0.44. Although both could be ok (type "String" will cast both them correctly at CB interaction) I'd suggest to use the same in both cases.

Reply:
Fixed in 27e54d40bca4fcf8961987beea741a1778f9791b